src/view/frontend/web/js/view/payment/method-renderer/paymentwall.js

```diff
@@ -38,6 +38,10 @@ define(
                 return JSON.parse(localStorage['mage-cache-storage'])['checkout-data']['validatedEmailValue'];
             },
 
+            getBillingValue: function () {
+                return JSON.stringify(JSON.parse(localStorage['mage-cache-storage'])['checkout-data']['billingAddressFromData']);
```

Comment:
Magento 2 already have Js API to get `localStorage.'
Please check `mage/storage`